news/atom.xml

```diff
@@ -14,8 +14,11 @@ layout:
     <name>The ev3dev team</name>
   </author>
 
+  {% assign index = 0 %}
```

Comment:
Because we need to filter the posts to get only news items. We want ten news items, not all the news items in the first ten posts. There are features in Jekyll 3.2 that would make this work ([the `where_exp` filter](https://github.com/jekyll/jekyll/pull/4478)), but GH pages is still on 3.1.

This applies to the second comment as well.